releasenotes/notes/fixwinprocessargs-4d745db8ff52e3fc.yaml

```diff
@@ -0,0 +1,16 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    On Windows, fixes Process agent memory leak when obtaining process arguments.
+other:
```

Comment:
I think you'll want to remove this section?

Reply:
why?